flink-runtime/src/main/scala/org/apache/flink/runtime/jobmanager/JobManager.scala

```diff
@@ -1114,7 +1114,8 @@ class JobManager(
           Option(jobGraph.getSerializedExecutionConfig()
```

Comment:
I think changing the default would impact folks in a rather surprising way when they migrate to flink 1.2 right. Let me know if you strongly believe that the default should be modified and the option is removed. Will change that.